spec/services/cfe/submission_manager_spec.rb

```diff
@@ -49,113 +49,122 @@ module CFE
-          allow(CreateApplicantService).to receive(:call).and_return(true)
-          allow(CreateCapitalsService).to receive(:call).and_return(true)
-          allow(CreatePropertiesService).to receive(:call).and_return(true)
-          allow(CreateVehiclesService).to receive(:call).and_return(true)
-          allow(CreateStateBenefitsService).to receive(:call).and_return(true)
-          allow(CreateDependantsService).to receive(:call).and_return(true)
-          allow(CreateOtherIncomeService).to receive(:call).and_return(true)
-          allow(CreateExplicitRemarksService).to receive(:call).and_return(true)
-          allow(ObtainAssessmentResultService).to receive(:call).and_return(true)
-          allow(CreateIrregularIncomesService).to receive(:call).and_return(true)
-          allow(CreateCashTransactionsService).to receive(:call).and_return(true)
+      before do
+        allow(CreateAssessmentService).to receive(:call).and_return(true)
+        allow(CreateApplicantService).to receive(:call).and_return(true)
+        allow(CreateCapitalsService).to receive(:call).and_return(true)
+        allow(CreatePropertiesService).to receive(:call).and_return(true)
+        allow(CreateVehiclesService).to receive(:call).and_return(true)
+        allow(CreateStateBenefitsService).to receive(:call).and_return(true)
+        allow(CreateDependantsService).to receive(:call).and_return(true)
+        allow(CreateOtherIncomeService).to receive(:call).and_return(true)
+        allow(CreateExplicitRemarksService).to receive(:call).and_return(true)
+        allow(ObtainAssessmentResultService).to receive(:call).and_return(true)
+        allow(CreateIrregularIncomesService).to receive(:call).and_return(true)
+        allow(CreateCashTransactionsService).to receive(:call).and_return(true)
+      end
+
+      context "with a passported application" do
+        let(:application) { create :legal_aid_application, :with_everything, :with_positive_benefit_check_result, :applicant_entering_means, vehicle: }
+
+        it "creates a submission record for the application" do
+          expect { submission_manager.call }.to change(application.cfe_submissions, :count).by(1)
         end
 
-        context "passported application" do
-          let(:application) { create :legal_aid_application, :with_everything, :with_positive_benefit_check_result, :applicant_entering_means, vehicle: }
+        it "calls expected services" do
+          expect(CreateAssessmentService).to receive(:call).once
+          expect(CreateProceedingTypesService).to receive(:call).once
+          expect(CreateApplicantService).to receive(:call).once
+          expect(CreateCapitalsService).to receive(:call).once
+          expect(CreatePropertiesService).to receive(:call).once
+          expect(CreateVehiclesService).to receive(:call).once
+          expect(CreateExplicitRemarksService).to receive(:call).once
+          expect(ObtainAssessmentResultService).to receive(:call).once
```

Comment:
Not a big deal, but it might be worth defining the passported/non-passported services and iterating them.

```ruby
let(:passported_services) do
  [
    CreateAssessmentService,
    CreateProceedingTypesService,
    ...
  ]
end

it "calls expected services" do
  passported_services.each do |service|
    allow(service).to receive(:call).and_return(true)
  end
  
  submission_manager.call
  
  passported_services.each do |service|
    expect(service).to have_received(:call).exactly(1).time
  end
end
```

I don't mind too much either way, but it might make it a bit more readable (my lazy eyes glaze over a see of `Service`s 😆.

What do you think?

Reply:
I did not change this except to ensure the `once`, but regardless I  find the above approach good in some respects but have had experience with that "iteration approach" making failures harder to resolve. 🤔

Reply:
have tidied up using a similar approach to that suggested, rubocop rspec suggest going further once that is in:

```
 expect(passported_services).to all(have_received(:call).once
```